shared/chat/conversation/messages/system-create-team/container.tsx

```diff
@@ -39,9 +39,10 @@ export default Container.connect(
   }),
   (stateProps, dispatchProps, ownProps: OwnProps) => ({
     creator: ownProps.message.creator,
-    isAdmin: stateProps.isAdmin,
+    isAdmin: stateProps.isAdmin || stateProps.you === ownProps.message.creator,
```

Comment:
what is `message.creator`?

Reply:
The person who created the team. For some reason isAdmin isn't accurate here, but the designs say to show if you created the team. Maybe should just change it to isCreator

Reply:
Or it could just use `isAdmin` if it's using the team role map, that should always be there.

Reply:
Hmm for some reason this didn't show up when I tested it before adding this though

Reply:
it was testing for admin instead of both admin and owner. Fixed.